www/package.json

```diff
@@ -49,10 +49,10 @@
     "url": "git+https://github.com/facebook/react.git"
   },
   "scripts": {
-    "dev": "gatsby develop",
+    "dev": "gatsby develop -H 0.0.0.0",
     "lint": "./node_modules/.bin/eslint --ext .js,.jsx --ignore-pattern public .",
     "test": "echo \"Error: no test specified\" && exit 1",
-    "develop": "gatsby develop",
+    "develop": "gatsby develop -H 0.0.0.0",
```

Comment:
Weird that we have both of these. That was an accident on my part. Let's just remove `develop` 😄